### PR TITLE
chore: organize and add details to plugins list

### PR DESCRIPTION
**Pull request by emilyrohrbough**
List Plugins in order:
- official
- verified
- community
- experimental

Added additional descriptions to direct users to our API docs / existing guides.

### CODE REVIEW

File: src/data/plugins.json [hunk 1]
```diff
@@ -361,8 +376,49 @@
     },
     {
       "name": "Custom Commands",
+      "description": " Read the <a target=\"_blank\" href=\"https://docs.cypress.io/api/plugins/preprocessors-api\">Custom Commands</a> and <a target=\"_blank\" href=\"https://docs.cypress.io/api/plugins/preprocessors-api\">Custom Query</a> documentation to learn more.",
```
Comment (AtofStryker) [inline src/data/plugins.json:379]:
```suggestion
      "description": " Read the <a target=\"_blank\" href=\"/api/plugins/preprocessors-api\">Custom Commands</a> and <a target=\"_blank\" href=\"/api/plugins/preprocessors-api\">Custom Query</a> documentation to learn more.",
```

**Review by AtofStryker — CHANGES_REQUESTED** [review]

File: src/data/plugins.json [hunk 2]
```diff
@@ -92,16 +93,79 @@
...
+        },
+        {
+          "name": "@cypress/instrument-cra",
+          "description": "NPM module for create-react-app applications to instrument source code without ejecting react-scripts.",
+          "link": "https://github.com/cypress-io/instrument-cra",
+          "keywords": ["coverage"],
+          "badge": "official"
+        },
+        {
+          "name": "Cypress Chrome Recorder",
```
[the remark this answers is not in the document]
Comment (emilyrohrbough) [reply src/data/plugins.json:139]: I don't think so. they were both listed before. I feel like the Chrome Reporter is what's shipped for the extension, but you can use it without adding the extension from reading the repo readme?

[the remark this answers is not in the document]
Comment (AtofStryker) [reply src/data/plugins.json:139]: It sounds like it lives in multiple places, which is kind of confusing. Maybe something for later 😅

File: src/data/plugins.json [hunk 3]
```diff
@@ -2,73 +2,74 @@
   "plugins": [
     {
       "name": "Preprocessors",
+      "description": "A preprocessor is the plugin responsible for preparing a support file or a test file for the browser. By default, Cypress comes packaged with webpack preprocessor already installed. Read the <a target=\"_blank\" href=\"https://docs.cypress.io/api/plugins/preprocessors-api\">Preprocessors API docs</a> to learn more.",
```
Comment (AtofStryker) [inline src/data/plugins.json:5]:
```suggestion
      "description": "A preprocessor is the plugin responsible for preparing a support file or a test file for the browser. By default, Cypress comes packaged with webpack preprocessor already installed. Read the <a target=\"_blank\" href=\"/api/plugins/preprocessors-api\">Preprocessors API docs</a> to learn more.",
```

**Review by AtofStryker — APPROVED** [review]
missed one absolute link on the first pass through. Besides that, looks good to me!

**Pull request merged**